pkg/ice/agent.go

```diff
@@ -172,7 +172,6 @@ func (a *Agent) pingCandidate(local, remote Candidate) {
 func (a *Agent) keepaliveCandidate(local, remote Candidate) {
 	msg, err := stun.Build(stun.ClassIndication, stun.MethodBinding, stun.GenerateTransactionId(),
 		&stun.Username{Username: a.remoteUfrag + ":" + a.LocalUfrag},
-		&stun.IceControlled{TieBreaker: a.tieBreaker},
 		&stun.Priority{Priority: uint32(local.GetBase().Priority(HostCandidatePreference, 1))},
```

Comment:
Is the priority attribute needed or does Janus only require the authorization? Otherwise we may as well clean it up now.

Reply:
Priority wasn't needed either, update PR! Thanks for catching that.

Reply:
I double checked, and every other value was asserted by libnice though